jobs/build/pre-release/Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def prerelease = load("../release/release.groovy")
+    def buildlib = prerelease.buildlib
+    def commonlib = prerelease.commonlib
+    def quay_url = "quay.io/openshift-release-dev/ocp-release-nightly"
+
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'FROM_RELEASE_TAG',
+                        description: 'Build tag to pull from (i.e. 4.1.0-0.nightly-2019-04-22-005054)',
```

Comment:
I would like this job to use the latest accepted when parameter is not specified (which would be most of the time). That way we can just run it on a schedule, and eventually, trigger on UMB -- no human providing parameters.

If that is hard in the short term, a human can fill the parameter for a while just to have something that works at all.

Reply:
I checked on https://api.ci.openshift.org/console/project/ocp/browse/images/release, they have a tag called `latest` but not sure if it's 4.1 or 4.2, I think it should be worth asking them to provide a tag for us.